.husky/pre-commit

```diff
@@ -1 +1 @@
-yarn dlx --no-install lint-staged
\ No newline at end of file
+yarn install
```

Comment:
This happened on its own when I ran `yarn` 🤔. Changed it back, but then it kept erroring that there is no `--no-install` option

Reply:
Yea i fixed it now